CONTRIBUTING.md

```diff
@@ -103,6 +103,9 @@ Push the local commit to GitHub fork repo:
 git push origin bugfix-branch:bugfix-branch
 ```
 
+Note that since GitHub requires submitting code through `username + token` (instead of using `username + password` directly), you need to create a GitHub token from https://github.com/settings/tokens:
```

Comment:
seems it's not necessary for normal user/dev to use this?

Reply:
mainly for new contributors